src/communication/network_interface_sr.c

```diff
@@ -10972,12 +10978,23 @@ sflashback_get_loginfo (THREAD_ENTRY * thread_p, unsigned int rid, char *request
   ptr = or_unpack_int (ptr, &context.num_loginfo);
   ptr = or_unpack_int (ptr, &context.forward);
 
+  FLASHBACK_SET_REQUEST_TIME ();
```

Comment:
IMHO, it would be nice to place at the position immediately after the function call.

Reply:
Yes, this will be used for check interval, so I will move it to the end of the function.